packages/instantsearch.js/src/components/SearchBox/SearchBox.tsx

```diff
@@ -183,6 +183,7 @@ class SearchBox extends Component<
             onInput={this.onInput}
             onBlur={this.onBlur}
             onFocus={this.onFocus}
+            aria-label="Search for results"
```

Comment:
is this text a guideline somewhere? personally I'd expect just "search", because what else than results can you search for?

Reply:
I'm happy to change this to just "Search", we had initially said "Search for products" ([APEX-713](https://algolia.atlassian.net/browse/APEX-713)), but I figured this component would be used for more than just products so went more general